test/e2e/common/runtime.go

```diff
@@ -189,7 +189,7 @@ while true; do sleep 1; done
 
 			/*
 				Release: v1.15
-				Name: Container Runtime, TerminationMessagePath, non-root user and non-default path
+				Testname: Container Runtime, TerminationMessagePath, non-root user and non-default path
```

Comment:
Confirmed this 4 parts are last ones what `Name:` instead of `Testname:` under test/e2e with `$ grep "\sName: " * -R | grep -v '"'`